### PR TITLE
Hello Microverse Project PR

### PR DESCRIPTION
**Pull request by carlosisraelsilva**
**In this project milestone, I did the following:**

- Added the `index.html` 
- Added the README.md file.
- Ensured there was no linter error.

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @carlosisraelsilva , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no readme file and a PR Description. Kindly document your work as it is a good practice.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by carlosisraelsilva** [comment]
Hi Frank!

I think there's something wrong with your review. 
I do submitted the README file and my PR does have a description "The New Pull Request" as asked on the previous review.


 image

image

**Comment by carlosisraelsilva** [comment]
add the README file

File: README.md [hunk 1]
```diff
@@ -1,32 +1,8 @@
...
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
```
Comment (zieeco) [inline README.md]: - Hey, great job adding the readme as requested by the previous reviewer, however, let's update the readme to the current readme [template](https://github.com/microverseinc/readme-template) and customize it to suit your project needs by removing some headers that are not relevant to this current project, however, those headers would be relevant to the subsequent project as your make progress in the program. Also, kindly follow the structure of the template in order to be consistent to make your readme more professional.

Comment (zieeco) [reply README.md]: ### PULL REQUEST TITLE

- Great job with the pull request title, however, the purpose of the project is to help you learn how to set up a professional Github repository and make a descriptive Pull Request with gives a summary of what you've done in a project.  You can further improve the Title by making it more descriptive e.g, `Hello Microverse Project`, as this is more descriptive as compared to `The New Pull Request `. Please, let's kindly update this.
---

### PULL REQUEST DESCRIPTION

- Great job so far setting up the `Hello Microverse Project`, however, you've not added any description that summarizes what you've done so far in the Pull Request.  The image below shows how to edit the Pull Request Description.

Screenshot from 2022-12-05 16-32-11

- You can add something like 

```
**In this project milestone, I did the following:**

- Added the `index.html` file with an `h1` tag.
- Added the `style.css` file.
- Ensured there was no linter error.

```

This is just a starting point on how to have a descriptive summary in your Pull Request. Please, let's try and update this little issue to have our project approved. :pray:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @carlosisraelsilva ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Great start. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @carlosisraelsilva 👋, 
image
In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**